### ISSUE TITLE
People copying your mod

### ISSUE THREAD

**Issue by MatadorProBr** [Herysia/CustomKeyBinds#6]
Well I came to talk about a serious matter, I always liked your mod (even if it is a mod that few people use) but to some people who are copying your mod giving no credit as for example a mod called MODDED, a mod that in reality I don’t even say that it’s a mod because it literally copies half of the mods and puts it on his mod without any credit like CrowdedMod that some prints he got from CrowdedMod and was saying that the CrowdedMod team help him put the mod on but in reality and everything lies about him:
![image](https://user-images.githubusercontent.com/71022656/112307219-72d18100-8c7f-11eb-978c-33077a98893a.png)
![image](https://user-images.githubusercontent.com/71022656/112307237-76fd9e80-8c7f-11eb-88b5-7a9c2e69a2e5.png)
![image](https://user-images.githubusercontent.com/71022656/112307247-795ff880-8c7f-11eb-8daf-072a9cda353b.png)
![image](https://user-images.githubusercontent.com/71022656/112307302-8bda3200-8c7f-11eb-90aa-0de8d95c0f1e.png)
Prints taken from the NodePolus server sent by przebot

Print the part he copied the mod from:
![image](https://user-images.githubusercontent.com/71022656/112307432-b0360e80-8c7f-11eb-87af-2b2af0514d1d.png)

Prints of the mod "made" by Boss_GamerYT:
![image](https://user-images.githubusercontent.com/71022656/112308258-b678ba80-8c80-11eb-9c72-08216a6cbb00.png)
![image](https://user-images.githubusercontent.com/71022656/112308267-b8db1480-8c80-11eb-8f0e-5739c742b3c5.png)
![image](https://user-images.githubusercontent.com/71022656/112308277-bb3d6e80-8c80-11eb-95dd-1a076c57262a.png)
![image](https://user-images.githubusercontent.com/71022656/112308287-bd9fc880-8c80-11eb-8dd8-502e880998ab.png)
![image](https://user-images.githubusercontent.com/71022656/112308290-c0022280-8c80-11eb-9fda-a8064cf95f53.png)
![image](https://user-images.githubusercontent.com/71022656/112308300-c2647c80-8c80-11eb-9f3d-84e82a5e5d2e.png)
![image](https://user-images.githubusercontent.com/71022656/112308309-c42e4000-8c80-11eb-8355-6ee462dfc4bb.png)
![image](https://user-images.githubusercontent.com/71022656/112308342-cdb7a800-8c80-11eb-8a31-02f6f592dfb1.png)
![image](https://user-images.githubusercontent.com/71022656/112308357-d01a0200-8c80-11eb-9a0d-e0ab01117989.png)
![image](https://user-images.githubusercontent.com/71022656/112308366-d1e3c580-8c80-11eb-9de2-c78d47b7edfa.png)
![image](https://user-images.githubusercontent.com/71022656/112308373-d314f280-8c80-11eb-98da-d8d6b36c4789.png)
![image](https://user-images.githubusercontent.com/71022656/112308387-d60fe300-8c80-11eb-86a4-5a919ab393b8.png)

Discord server: https://discord.gg/dRMNZa7rcM

Link of GitHub: https://github.com/Bossgamerteam/100-player-among-us-mod

How to install tutorial by one of the server Administrators: https://youtu.be/ZG4hgUo7EBM

I sincerely just wanted you to talk to him on Discord, his Discord is: Boss_GamerYT# 0001
He will probably lie at first but then he will accept it and give credit

**Comment by przebor** [comment]
I agree that we have to do something with this. "EXTREME GAMER" is not respecting licenses nor our work. 
He only added a little note "Mod made by CrowdedMods" after I told him that he has to do that (license).

**Comment by przebor** [comment]
Herysia is now fully able to submit DMCA infringement complaint to either Discord or Github.
I think we should also talk to them in Discord group chat first. @Herysia please notify me when you reply, we can collaborate on taking down license violators.

**Comment by MatadorProBr** [comment]
I can't take this guy anymore copying the mods and earning all the credits and still saying that the mod has "support" for mobile, as it does not modify the client side or the server (the server it just puts plugins that are also not made by him and yes plugins that people who made and put as open source on GitHub) I never thought you would talk about this issue lol

**Comment by MatadorProBr** [comment]
I found another proof that he is copying his mod without giving any credit:
![image](https://user-images.githubusercontent.com/71022656/112508263-29ac2a80-8d6e-11eb-9040-9176b4223f66.png)
(NOTE: He only gave it to CrowdedMod for the explanation above)
